Resources/public/js/views/ez-editactionbarview.js

```diff
@@ -99,6 +113,19 @@ YUI.add('ez-editactionbarview', function (Y) {
                 setter: function (val, name) {
                     this._setVersion(val);
                 }
+            },
+
+            /**
+             * The language code of the content currently edited
+             *
+             * @attribute languageCode
+             * @type {String}
+             */
+            languageCode: {
+                value: {},
```

Comment:
same as https://github.com/ezsystems/PlatformUIBundle/pull/385/files#r42241825 (and missing the @default tag in the JS Doc)